### PR TITLE
Created  basic layout structure for main page

### PR DESCRIPTION
**Pull request by Lokeshranjan8**
This PR introduces a basic layout structure for the main page of the CatGPT application.

Changes include:
- Created a new `MainLayout` component with placeholders for the chat area and user input section  .
- Separated CSS into `MainLayout.css` for better maintainability.
- Updated `App.tsx` to integrate `MainLayout`.
catGPT

### CODE REVIEW

File: client/src/MainLayout.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.main-layout {
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+}
```
Comment (Sweetdevil144) [inline client/src/MainLayout.css:5]: Ahaa. Please use TailwindCSS instead of Pure CSS for better UI

File: client/src/MainLayout.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './MainLayout.css';
+
+const MainLayout: React.FC = () => {
+    return (
+        <div className="main-layout">
+            <header className="layout-header">
+                <h1>cat-GPT</h1>
```
Comment (Sweetdevil144) [inline client/src/MainLayout.tsx:8]: Cat-GPT

File: client/src/MainLayout.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+            </header>
+            <main className='chat-sec'>
+                <div>
+                    {/* Chat Area Placeholder */}
+                    <span>CHAT AREA</span>
```
Comment (Sweetdevil144) [inline client/src/MainLayout.tsx:13]: Can you place a dummy Cat GIF !!

**Review by Sweetdevil144 — COMMENTED** [review]
Overall, vanilla CSS was not preferred. Can you revert back the same to using the configures TailwindCSS. Add configurations too in addition if you like so.

File: client/src/MainLayout.css (hunk 1 above)
Comment (Lokeshranjan8) [reply client/src/MainLayout.css:5]: sure bhaiya-

File: client/src/MainLayout.tsx (hunk 3 above)
Comment (Lokeshranjan8) [reply client/src/MainLayout.tsx]: catGPT
is this good?

**Review by Sweetdevil144 — APPROVED** [review]
LGTM

**Pull request merged**